src/test/ui/consts/issue-73976-monomorphic.rs

```diff
@@ -17,7 +18,9 @@ impl<T: 'static> GetTypeId<T> {
 }
 
 const fn check_type_id<T: 'static>() -> bool {
-    matches!(GetTypeId::<T>::VALUE, GetTypeId::<usize>::VALUE)
+    // FIXME(const_cmp_type_id)
+    //GetTypeId::<T>::VALUE == GetTypeId::<usize>::VALUE
+    true
 }
```

Comment:
Should this be fixed before landing?

Reply:
Oh, is that all handled by this?
* #101698

Reply:
Yes with only this PR and not https://github.com/rust-lang/rust/pull/101698 it is impossible to do this test.